app/assets/stylesheets/app/product_list.scss

```diff
@@ -0,0 +1,7 @@
+.product_list_container{
+    display: flex;
+    justify-content: space-between;
+}
+.product_list{
+    flex: 0 0 calc(25% - 2em);
```

Comment:
Why Flexbox? CSS Grid gives even spacing. Consider e.g. `grid: auto-flow / repeat(4, 1fr);`, which relieves you of this `calc` statement.

Reply:
It doesn't appear that tablexi uses grid anywhere in the project. I think in order for good browser we are limited to flex.

Reply:
Oh. Right!